build.sh

```diff
@@ -24,7 +24,7 @@ $EM_DIR/emconfigure ./configure \
     --disable-workshop \
     --disable-netbeans \
     --disable-sniff \
-    --disable-multibyte \
+    --enable-multibyte \
```

Comment:
Have you tested this?
How much would the size of `vim.sj` be increased? And are the characters displayed correctly?